.github/workflows/integer_full_benchmark.yml

```diff
@@ -107,6 +107,12 @@ jobs:
         run: |
           make BENCH_OP_FLAVOR=${{ matrix.op_flavor }} bench_${{ matrix.command }}
 
+      # Run these benchmarks only once
+      - name: Run OPRF benchmarks
+        if: matrix.command == 'integer' && matrix.op_flavor == 'default'
+        run: |
+          make bench_integer_oprf
```

Comment:
could not put the oprf in the same benchmarks as the integers ?

sorry just realized now

Reply:
You mean like the cast benchmarks in shortint ?

Reply:
what do you mean ?

Reply:
What do YOU mean then :smirk: ?
When you say `put the oprf in the same benchmark as the integers` . At the workflow level ? Makefile level ? Rust benchmark file level ?

Reply:
rust benchmark, sorry wasn't clear